### PR TITLE
Add support for export service

### PR DESCRIPTION
**Pull request by pazaan**
Add commented-out support for the data export service to `docker-compose.yml`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,6 +189,34 @@ Comment out the entire `mongo:` section in the `docker-compose.yml` file. For ex
   #     - '27017:27017'
 ```
 
+### Running a Mongo replica set
```
Comment (darinkrauss) [inline README.md]: Why put this right in the middle of the ## Alternate Mongo Host section? 

You should think about cloning the format of the entire ## Alternate Mongo Host, except updating the instructions for ## Running a Mongo Replicate Set.

Comment (darinkrauss) [reply README.md]: BTW, note the number of hashes.

File: .env [hunk 2]
```diff
@@ -146,3 +146,7 @@ TIDEPOOL_DOCKER_TIDE_WHISPERER_PORT_PREFIX=
 
 # viz
 TIDEPOOL_DOCKER_VIZ_DIR=viz
+
+# export-service
```
Comment (darinkrauss) [inline .env]: Isn't "export service" redundant? Frankly, I'd get rid of the whole "-service" thing everywhere for consistency. None of our other services are called "-service".

Don't you want to be able to work on it locally? If so, you'll need the `_DIR` environment variables.

File: docker-compose.yml [hunk 3]
```diff
@@ -554,5 +556,17 @@ services:
   #     - '8081:8081'
   #     - '8082:8082'
 
+  # data-export-service:
+  #   image: tidepool/data-export-service
+  #   ports:
+  #     - '9300-9301:9300-9301'
+  #   environment:
+  #     HTTPS_CONFIG: '{"cert": "${TIDEPOOL_DOCKER_STYX_TLS_CERTIFICATE}", "key": "${TIDEPOOL_DOCKER_STYX_TLS_PRIVATE_KEY}"}'
```
Comment (darinkrauss) [inline docker-compose.yml]: We don't run SSL locally.

No build or volumes sections?

You can drop the single quotes around all of these values.

What is the SESSION_SECRET? Also, as you can see from the rest of the file, we've put pretty much everything in external environment variables.

File: README.md (hunk 1 above)
Comment (pazaan) [reply README.md]: I'll just remove it for now. I'm not sure what the full implications are, anyway (services that won't reconnect, etc).

File: .env (hunk 2 above)
Comment (pazaan) [reply .env]: Yeah "service" is redundant. I'll fix that.

File: docker-compose.yml (hunk 3 above)
Comment (pazaan) [reply docker-compose.yml]: Good point - that was for some local testing I was doing.
Have documented SESSION_SECRET in the upcoming `.env` push :)

**Comment by pazaan** [comment]
@darinkrauss can you re-review after latest changes?

File: docker-compose.yml [hunk 4]
```diff
@@ -554,5 +556,19 @@ services:
...
+  # export:
+  #   image: tidepool/export
+  #   # build: ${TIDEPOOL_DOCKER_EXPORT_DIR}
+  #   # volumes:
+  #   #   - ${TIDEPOOL_DOCKER_EXPORT_DIR}:/app:cached
+  #   #   - /app/node_modules
+  #   ports:
+  #     - '${TIDEPOOL_DOCKER_EXPORT_PORT_PREFIX}9300:${TIDEPOOL_DOCKER_EXPORT_PORT_PREFIX}9300'
+  #   environment:
+  #     API_HOST: http://${TIDEPOOL_DOCKER_STYX_HOST}:8009
```
Comment (darinkrauss) [inline docker-compose.yml]: Not that it really matters, but the environment variables in the rest of the services are alphabetically ordered (for finding quickly).

**Pull request merged**